astropy/convolution/convolve.py

```diff
@@ -374,14 +375,12 @@ def convolve(array, kernel, boundary='fill', fill_value=0.,
         return result
 
 
-@deprecated_renamed_argument('interpolate_nan', 'nan_treatment', 'v2.0.0')
 @support_nddata(data='array')
 def convolve_fft(array, kernel, boundary='fill', fill_value=0.,
                  nan_treatment='interpolate', normalize_kernel=True,
                  normalization_zero_tol=1e-8,
                  preserve_nan=False, mask=None, crop=True, return_fft=False,
-                 fft_pad=None, psf_pad=None, quiet=False,
```

Comment:
`quiet` is not used for anything, so I've removed it. Maybe it needs to be mentioned in the changelog, too?

Reply:
Since it wasn't doing anything... maybe not?

Reply:
The only issue could be if someone was using it as a positional argument that now disappears.
Btw, I'm adding the "use kwarg only arguments" to the discussion list for the meeting, I think this function (among many others) could benefit by it.